domain/status/state/state.go

```diff
@@ -40,6 +41,40 @@ func NewState(factory database.TxnRunnerFactory, clock clock.Clock, logger logge
 	}
 }
 
+// GetModelInfo returns the model's basic information.
+func (st *State) GetModelInfo(ctx context.Context) (status.ModelStatusInfo, error) {
+	type modelInfo struct {
```

Comment:
it is our convention to place these in the types.go file for this domain. We are not trying to inline db structs anymore.

Reply:
I see, sure thanks tom!